portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,23 +25,34 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> comments_ = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> comments = new ArrayList<String>();
-    comments.add("This page looks great");
-    comments.add("nice job");
-    comments.add("perfect");
     String json = "{";
-    json += "\"commentOne\": ";
-    json += "\"" + comments.get(0) + "\"";
-    json += ", ";
-    json += "\"commentTwo\": ";
-    json += "\"" + comments.get(1) + "\"";      
-    json += ", ";
-    json += "\"commentThree\": ";
-    json += "\"" + comments.get(2) + "\""; 
-    json += "}";
+    for(int x=0; x<comments_.size(); x++){
+      json += "\"comment"+(x+1)+"\": ";
+      json += "\"" + comments_.get(x) + "\"";
+      if((comments_.size()-x)!=1){
+        json += ", ";}
+    }
```

Comment:
nit: indent this 2 spaces so it lights up with the "if", it just makes it easier to read.